easybuild/easyconfigs/x/xpdf/xpdf-4.04-GCCcore-12.3.0.eb

```
@@ -0,0 +1,38 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2023/11
+easyblock = 'CMakeMake'
+
+name = 'xpdf'
+version = '4.04'
+
+homepage = 'https://www.xpdfreader.com/'
+description = """Xpdf was first released in 1995. It was written, and is still developed, by     
+Derek Noonburg.
+Xpdf is a free PDF viewer and toolkit, including a text extractor, image        
+converter, HTML converter, and more. Most of the tools are available as open source."""
+
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = ['https://dl.xpdfreader.com/']
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['63ce23fcbf76048f524c40be479ac3840d7a2cbadb6d1e0646ea77926656bade']
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('binutils', '2.40'),
+]
+
+dependencies = [
+    ('zlib', '1.2.13'),
+    ('freetype', '2.13.0'),
+    ('libpng', '1.6.39'),
+    ('Qt5', '5.15.10'),
+    ('LittleCMS', '2.15')
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['bin'],
+}
```

we should really check that `bin/xpdf` exists explicitly. It's generally nice to define a sanity check command as well, perhaps something like this:
```suggestion
sanity_check_paths = {
    'files': ['bin/%(name)s'],
    'dirs': [],
}

sanity_check_commands = ["pdfinfo --help 2>&1 | grep 'pdfinfo version %(version)s'"]
```